projects/1.16/assets/macaws-roofs/mcwroofs/lang/zh_cn.json

```
@@ -1 +1,609 @@
+  "block.mcwroofs.oak_top_roof":"橡木顶端屋顶",
+  "block.mcwroofs.oak_three_way_roof":"橡木三向屋顶",
+  "block.mcwroofs.oak_four_way_roof":"橡木四向屋顶",
+  "block.mcwroofs.oak_lower_roof":"橡木低屋顶",
+  "block.mcwroofs.oak_steep_roof":"橡木陡峭屋顶",
+  "block.mcwroofs.spruce_roof":"云杉屋顶",
+  "block.mcwroofs.spruce_attic_roof":"云杉天窗屋顶",
+  "block.mcwroofs.spruce_cube_roof":"云杉方块屋顶",
+  "block.mcwroofs.spruce_top_roof":"云杉顶端屋顶",
+  "block.mcwroofs.spruce_three_way_roof":"云杉三向屋顶",
+  "block.mcwroofs.spruce_four_way_roof":"云杉四向屋顶",
+  "block.mcwroofs.spruce_lower_roof":"云杉低屋顶",
+  "block.mcwroofs.spruce_steep_roof":"云杉陡峭屋顶",
+  "block.mcwroofs.birch_roof":"白桦屋顶",
+  "block.mcwroofs.birch_attic_roof":"白桦天窗屋顶",
+  "block.mcwroofs.birch_cube_roof":"白桦方块屋顶",
+  "block.mcwroofs.birch_top_roof":"白桦顶端屋顶",
+  "block.mcwroofs.birch_three_way_roof":"白桦三向屋顶",
+  "block.mcwroofs.birch_four_way_roof":"白桦四向屋顶",
+  "block.mcwroofs.birch_lower_roof":"白桦低屋顶",
+  "block.mcwroofs.birch_steep_roof":"白桦陡峭屋顶",
+  "block.mcwroofs.jungle_roof":"丛林屋顶",
+  "block.mcwroofs.jungle_attic_roof":"丛林天窗屋顶",
+  "block.mcwroofs.jungle_cube_roof":"丛林方块屋顶",
+  "block.mcwroofs.jungle_top_roof":"丛林顶端屋顶",
+  "block.mcwroofs.jungle_three_way_roof":"丛林三向屋顶",
+  "block.mcwroofs.jungle_four_way_roof":"丛林四向屋顶",
+  "block.mcwroofs.jungle_lower_roof":"丛林低屋顶",
+  "block.mcwroofs.jungle_steep_roof":"丛林陡峭屋顶",
+  "block.mcwroofs.acacia_roof":"金合欢屋顶",
+  "block.mcwroofs.acacia_attic_roof":"金合欢天窗屋顶",
+  "block.mcwroofs.acacia_cube_roof":"金合欢方块屋顶",
+  "block.mcwroofs.acacia_top_roof":"金合欢顶端屋顶",
+  "block.mcwroofs.acacia_three_way_roof":"金合欢三向屋顶",
+  "block.mcwroofs.acacia_four_way_roof":"金合欢四向屋顶",
+  "block.mcwroofs.acacia_lower_roof":"金合欢低屋顶",
+  "block.mcwroofs.acacia_steep_roof":"金合欢陡峭屋顶",
+  "block.mcwroofs.dark_oak_roof":"深色橡木屋顶",
+  "block.mcwroofs.dark_oak_attic_roof":"深色橡木天窗屋顶",
+  "block.mcwroofs.dark_oak_cube_roof":"深色橡木方块屋顶",
+  "block.mcwroofs.dark_oak_top_roof":"深色橡木顶端屋顶",
+  "block.mcwroofs.dark_oak_three_way_roof":"深色橡木三向屋顶",
+  "block.mcwroofs.dark_oak_four_way_roof":"深色橡木四向屋顶",
+  "block.mcwroofs.dark_oak_lower_roof":"深色橡木低屋顶",
+  "block.mcwroofs.dark_oak_steep_roof":"深色橡木陡峭屋顶",
+  "block.mcwroofs.crimson_roof":"绯红屋顶",
+  "block.mcwroofs.crimson_attic_roof":"绯红天窗屋顶",
+  "block.mcwroofs.crimson_cube_roof":"绯红方块屋顶",
+  "block.mcwroofs.crimson_top_roof":"绯红顶端屋顶",
+  "block.mcwroofs.crimson_three_way_roof":"绯红三向屋顶",
+  "block.mcwroofs.crimson_four_way_roof":"绯红四向屋顶",
+  "block.mcwroofs.crimson_lower_roof":"绯红低屋顶",
+  "block.mcwroofs.crimson_steep_roof":"绯红陡峭屋顶",
+  "block.mcwroofs.warped_roof":"诡异屋顶",
+  "block.mcwroofs.warped_attic_roof":"诡异天窗屋顶",
+  "block.mcwroofs.warped_cube_roof":"诡异方块屋顶",
+  "block.mcwroofs.warped_top_roof":"诡异顶端屋顶",
+  "block.mcwroofs.warped_three_way_roof":"诡异三向屋顶",
+  "block.mcwroofs.warped_four_way_roof":"诡异四向屋顶",
+  "block.mcwroofs.warped_lower_roof":"诡异低屋顶",
+  "block.mcwroofs.warped_steep_roof":"诡异陡峭屋顶",
+  "block.mcwroofs.oak_planks_roof":"橡木板屋顶",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "block.mcwroofs.oak_planks_roof":"橡木木板屋顶",
```
把这个修正之后，你检查一下其他的macaw系列，看还有没有这种木板/原木作为材料的家具。
如果是木板做的，就 xx木板xx，比如橡木木板椅子。
如果是原木做的，就 xx原木xx。
如果有改动，修改后整合在一个 PR 里。麻烦了。
你把这个改完之后我会merge，merge完你再改吧。

---

已修正。
其他macaw系列正在检查。